src/indicators/flooding/glofas/utils.py

```diff
@@ -108,7 +106,6 @@ def _convert_dict_to_ds(ds_glofas_dict) -> xr.Dataset:
         .rename({"step": "leadtime"})
     )
 
-
```

Comment:
Sorry to be pedantic but there should be two blank lines between methods (I only noticed because PyCharm complains!)